ui/raidboss/data/triggers/shinryu-ex.js

```diff
@@ -33,115 +37,197 @@
       regex: /:Shinryu starts using Akh Morn on (\y{Name})/,
       alertText: function(data, matches) {
         if (matches[1] == data.me)
-          return 'Akh Morn on you';
+          return {
+            en: 'Akh Morn on you',
+            de: 'Akh Morn auf dir',
+          };
         if (data.role == 'tank')
-          return 'Akh Morn on ' + matches[1];
+          return {
+            en: 'Akh Morn on ' + matches[1],
+            de: 'Akh Morn auf ' + matches[1],
+          };
       },
       infoText: function(data, matches) {
         if (matches[1] == data.me || data.role == 'tank')
           return;
-        return 'Akh Rhai: spread and move';
+        return {
+          en: 'Akh Rhai: spread and move',
+          de: 'Akh Rhai: Verteilen und bewegen',
+        };
       },
       tts: 'akh morn',
     },
     { id: 'ShinryuEx Diamond Dust',
       regex: /:25DD:Shinryu starts using Diamond Dust/,
-      infoText: function(data) { return 'Ice: Stack and Stop'; },
+      regexDe: /:25DD:Shinryu starts using Diamantenstaub/,
+      infoText: function(data) { return {
```

Comment:
Helpful hint: You only need function(data) or function(data, matches) if that function uses data or matches.  If it's just going to return a fixed dictionary of strings, it can just be `infoText: { en: 'something', de: 'something' },`